README.md

```diff
@@ -4,6 +4,9 @@
 
 > This is Florin Voicu portfolio, created with HTMl, CSS and Linters and will serve the purpose to get higher chances of employment.
 
+## Live demo
+
+https://vflorin1995.github.io/
```

Comment:
#### OPTIONAL

- [ ] Kindly use this format for your link `[live demo](https://vflorin1995.github.io/)` so it appears like this  [live demo](https://vflorin1995.github.io/) 👍🏿